ml_interview_concept_mindmap.Rmd

```diff
@@ -0,0 +1,10 @@
+## Explanation for the community contribution project

```

Comment:
The first line should start with a single hashtag #, followed by a single whitespace, and then the title.